Makefile

```diff
@@ -1,6 +1,8 @@
 # --- Global -------------------------------------------------------------------
 O = out
 COVERAGE = 91
+SEMVER ?= v0.0.0-dev
```

Comment:
we can use `$(shell git describe --tags --dirty)` here. When run on master, it outputs `v0.0.12`. When run on my `use-lint-action` branch, which branched before the last merge to master, it outputs `v0.0.11-1-g0ed4f98`. If you have uncommitted changes, it will append `-dirty` (although not if you have new files not added). This way the semver has some meaning on dev builds.

Reply:
that is very cool. thank you.